test/unit/replace_version_test.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+from exasol.toolbox.tools import replace_version
+
+
+def test_retrieve_issue_templates():
+    lines = ["first/line\n", "second/line@0.0.0\n",""]
+    expected = ["first/line\n", "second/line@9.9.9\n",""]
```

Comment:
also consider a "negative" test, where input == output because nothing needs to be changed.